### DISCUSSION TITLE
An error while trainning

### DISCUSSION THREAD

**Discussion by YoaviA** [talmolab/sleap#1468] [Help!]
The error is: tensorflow.python.framework.errors_impl.ResourceExhaustedError:  OOM when allocating tensor with shape[4,96,600,800] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc
I'm running Sleap on Windows, and trainning using a one-animal-model

**Comment by roomrys** [comment]
Hi @YoaviA,

OOM indicates an Out of Memory error. To solve this, you can try:
1. __If your video has higher resolution than needed__, decreasing the input scaling (we want the down sampled image to include at least 2pixels wide/tall for your smallest feature/node being tracked)
2. Switching to SLEAP's most memory efficient model: top-down, and setting the max number of instance to predict: 1.

Let us know if either of those help you!

Thanks,
Liezl
